realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -361,7 +361,7 @@ private static void initializeRealm(Realm realm) {
             }
         } catch (Exception e) {
             commitChanges = false;
-            throw e;
+            throw new RuntimeException(e);
```

Comment:
I don't think we need to wrap the exception since all possible exceptions are a subclass of `RuntimeException`.

I prefer catching `RuntimeException` instead of `Exception`.